src/commands/validate.io.ts

```diff
@@ -1,10 +1,11 @@
 import {
-  getRuleForResult,
+  getRuleForResult, //@TODO migrate to v2 method
```

Comment:
You missed a TODO here

Reply:
Done 👍 I reworked the code to correctly support multiple runs from single output for V2 method (even though we don't really have any case yet where output will contain more than one AFAIR).